src/twisted/python/compat.py

```diff
@@ -40,6 +40,16 @@
 else:
     _PY3 = True
 
+if sys.version_info >= (3, 4, 0):
+    _PY34PLUS = True
+else:
+    _PY34PLUS = False
+
+if sys.version_info >= (3, 5, 0):
+    _PY35PLUS = True
+else:
+    _PY35PLUS = False
+
```

Comment:
I don't feel strongly about that either way, but the one-liners seem nicer, is all.  :-)